WordPress/Classes/ViewRelated/Post/PostPreviewViewController.m

```diff
@@ -112,21 +111,22 @@ - (void)setupWebView
     [self.view addSubview:self.webView];
 }
 
-#pragma mark - Loading
+#pragma mark - Loading Animations
 
-- (void)startLoading
+- (void)startLoadAnimation
 {
     [self.navigationItem setLeftBarButtonItem:[self statusButtonItem] animated:YES];
     self.navigationItem.title = nil;
 }
 
-- (void)stopLoading
+- (void)stopLoadAnimation
```

Comment:
I think part of the issue we had is that the previous naming for these methods was confusing.  These methods now have a clearer naming which specifically relates to the loading UI changes / animations.  The idea is to better separate responsibilities.

Any piece of code that wants to make the web view stop loading, can do so directly.

Reply:
❤️ Thanks for renaming that!